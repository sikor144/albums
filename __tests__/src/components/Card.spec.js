import React from 'react';
import renderer from 'react-test-renderer';

import Card from '../../../src/components/Card';

describe('Card component', () => {
  it('renders component correctly', () => {
    const component = renderer.create(
      <Card />,
    ).toJSON();

    expect(component).toMatchSnapshot();
  });
});
