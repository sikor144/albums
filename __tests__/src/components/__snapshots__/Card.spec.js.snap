// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Card component renders component correctly 1`] = `
<View
  style={
    Object {
      "borderBottomWidth": 0,
      "borderColor": "#ddd",
      "borderRadius": 2,
      "borderWidth": 1,
      "elevation": 1,
      "marginLeft": 5,
      "marginRight": 5,
      "marginTop": 10,
      "shadowColor": "#000",
      "shadowOffset": Object {
        "height": 2,
        "width": 0,
      },
      "shadowOpacity": 0.1,
      "shadowRadius": 2,
    }
  }
/>
`;
